src/renderer/wallet/main/toolbar.jsx

```diff
@@ -207,44 +207,50 @@ const ProfileList = withStyles(profileStyle)(
 		onProfileSelect,
 		onClickCorporate,
 		closeProfile,
+		selectedProfile,
 		showCorporate
 	}) => {
 		return (
 			isOpen && (
 				<ClickAwayListener onClickAway={closeProfile}>
 					<div className={classes.profile}>
 						{profiles &&
-							profiles.map((el, index) => (
-								<Grid
-									container
-									key={index}
-									className={classes.profileDetail}
-									onClick={onProfileSelect(el)}
-								>
-									<Grid item sm={2}>
-										{el.type === 'corporate' ? (
-											<SmallRoundCompany />
-										) : el.profilePicture ? (
-											<HexagonAvatar
-												src={el.profilePicture}
-												className={classes.profileListAvatar}
-												smallAvatar={true}
-											/>
-										) : (
-											<SmallRoundPerson />
-										)}
+							profiles.map((el, index) =>
```

Comment:
I think It's more legible and clean filtering the array before executing the map
`profiles.filter(el => el.id !== selectedProfile.id).map(...`